drivers/sensors/cirque_pinnacle.c

```diff
@@ -204,40 +201,89 @@ void cirque_pinnacle_init(void) {
 #endif
 
     touchpad_init = true;
+
     // Host clears SW_CC flag
     cirque_pinnacle_clear_flags();
 
-    // Host configures bits of registers 0x03 and 0x05
-    RAP_Write(SYSCONFIG_1, SYSCONFIG_1_VALUE);
-    RAP_Write(FEEDCONFIG_2, FEEDCONFIG_2_VALUE);
-
-    // Host enables preferred output mode (absolute)
-    RAP_Write(FEEDCONFIG_1, FEEDCONFIG_1_VALUE);
+    // SysConfig1 (Low Power Mode)
+    // Bit 0: Reset, 1=Reset
+    // Bit 1: Shutdown, 1=Shutdown, 0=Active
+    // Bit 2: Sleep Enable, 1=low power mode, 0=normal mode
+    // send a RESET command now, in case QMK had a soft-reset without a power cycle
+#if CIRQUE_PINNACLE_RESET_ON_INIT_WAIT > 0
+    RAP_Write(SYSCONFIG_1, 0x01);
+    wait_ms(CIRQUE_PINNACLE_RESET_ON_INIT_WAIT);
+#endif
+    RAP_Write(SYSCONFIG_1, 0x00);
+    wait_us(50);
 
-    // Host sets z-idle packet count to 5 (default is 30)
-    RAP_Write(Z_IDLE_COUNT, Z_IDLE_COUNT_VALUE);
+    // FeedConfig2 (Feature flags for Relative Mode Only)
+    // Bit 0: IntelliMouse Enable, 1=enable, 0=disable
+    // Bit 1: All Taps Disable, 1=disable, 0=enable
+    // Bit 2: Secondary Tap Disable, 1=disable, 0=enable
+    // Bit 3: Scroll Disable, 1=disable, 0=enable
+    // Bit 4: GlideExtend® Disable, 1=disable, 0=enable
+    // Bit 5: reserved
+    // Bit 6: reserved
+    // Bit 7: Swap X & Y, 1=90° rotation, 0=0° rotation
+    RAP_Write(FEEDCONFIG_2, 0x00);
```

Comment:
Before it was `0x1C` so now the following features are enabled?

* Secondary Tap
* Scroll
* GlideExtend®

Or does this not have any effect on absolute mode at all? Also this screams for a nice bitfield in the future :slightly_smiling_face:

Reply:
Correct - all these features only apply to Cirque relative mode - which is not being used (and only half-way implemented).